src/lib/utils/__tests__/time.test.ts

```diff
@@ -0,0 +1,43 @@
+import { secondsToMidString, secondsToString } from "lib/utils/time";
+
+const minute = 60;
+const hour = 60 * minute;
+const day = hour * 24;
+
+describe("time", () => {
+  it("secondsToString", () => {
+    expect(secondsToString(0)).toBe("0secs");
+
+    expect(secondsToString(minute - 1)).toBe("59secs");
+    expect(secondsToString(minute)).toBe("1min");
+    expect(secondsToString(minute + 1)).toBe("2mins");
+
+    expect(secondsToString(hour - 1)).toBe("60mins");
+    expect(secondsToString(hour)).toBe("1hr");
+    expect(secondsToString(hour + 1)).toBe("2hrs");
+
+    expect(secondsToString(day - 1)).toBe("24hrs");
```

Comment:
same thing here I Think here it should be 23hours59mins instead of 24hours to stay consistent with the form of the code you wrote